tests/test_unitxt.py

```diff
@@ -0,0 +1,20 @@
+# First Party
+from instructlab.eval.unitxt import UnitxtEvaluator
+
+
+def test_unitxt():
+    print("===> Executing 'test_unitxt'...")
+    try:
+        model_path = "instructlab/granite-7b-lab"
+        unitxt_recipe = "card=cards.wnli,template=templates.classification.multi_class.relation.default,max_train_instances=5,loader_limit=20,num_demos=3,demos_pool_size=10"
+        unitxt = UnitxtEvaluator(model_path=model_path, unitxt_recipe=unitxt_recipe)
+        overall_score, single_scores = unitxt.run()
+        print(overall_score)
```

Comment:
I think the unitest should check the overall_score and see that they make sense.  This will ensure the api does not break at some point (E.g .returns 'none').